src/main/resources/sessionLinksRecoveryEmailTemplate-feedbackSessionAccessLinksNone.html

```diff
@@ -0,0 +1,31 @@
+<p>Hello,</p>
+
+<p>
+    You are receiving this email because a request to resend TEAMMATES links related to this email address <strong>${userEmail}</strong> was made on <a href="${teammateHomePageLink}">TEAMMATES</a>.
+</p>
+
+<p>
+    You do not have any feedback sessions for the past 180 days. If you have any doubt, please contact your course instructors to check if the email below is the one used for your course enrollment:
```

Comment:
`We could not find any sessions associated with this email address that have opened or closed in the past 180 days. `

`If a course that you expected to see is missing, it may be because it is associated with a different email address. In that case, you can <a  href=${sessionsRecoveryLink}>try again</a> with a different email address or contact your course instructor(s) to check whether this email address is the one used to enroll you in TEAMMATES. `
`Also, sessions that have been not opened or closed in the past 180 days are not shown. If you want the links to those sessions, contact the TEAMMATES support team mentioned below.`

You can remove the rest (upto `Technical help...`)